File: C++/Stack.cpp
#include<iostream>
using namespace std;
#define size 5
int top=-1;
int stack[size];
int push(int x){
	if(top==size-1){
		cout<<"Stack overflow";
		
	}
	else{
		top++;
		stack[top]=x;
	}
}

void pop(){
	if(top==-1){
		cout<<"Stack underflow";
	}
	else{
		int item=stack[top];
		top--;
	}
	
}
void display(){
	if(top==-1){
		cout<<"Stack is Empty";
		
	}
	else{
		for(int i=top;i>=0;i--){
			cout<<stack[i];
		}
	}
}
int main(){
	int choice,x;
	while(1){
		
	cout<<"\nENTER YOUR CHOICE :";
	cin>>choice;
	switch(choice){
		
		case 1:
			cout<<"\nENTER THE ELEMENT YOU WANT TO PUSH:";
			cin>>x;
			push(x);
			cout<<"\nTHE ELEMENT YOU PUSHED IS :"<<x;
			break;
			case 2:
				pop();
				break;
				case 3:
					display();
					break;
					case 4:
						exit(0);
						
	}
	}
	return 0;
}

File: Python/stack.py
# n=int(input())
# l=[]
# max=10
# top=0
# bottom=0
# def puspstack(n):
#     # if top!=bottom:
#     #     l.append(n)
#     #     top+=1
#     # else:
#     #    print("stack overflow")
#     if top==max:
#         print("stack is full")
#         return
#     l.append(n)
#     top+=1
            
# def popstack(n):
#     # if (top==bottom):
#     #     print("stack is underflow u cant pop")
#     # else:
#     #     top-=1
#     #     for i in range(max):
#     #         l.pop(n)
#     if top==bottom:
#         print("stack is empty")
#         return
#     print(f"{l[top-1]} is poped out")
#     top-=1
# def printstack():
#     if not l:
#         print("stack is empty")
#         return
#     for i in range(top-1,-1,-1):
#         print(l[i])
# printstack()
# puspstack()
# puspstack()
# puspstack()
# printstack()
# popstack()
    
class stack:
    list=[]
    def __init__(self,max):
        self.top=0
        self.bottom=0
        self.max=5
    def pushstack(self):
        if self.top==self.max:
            return  f"stack is overflow"
        elif(self.top!=self.max):
            self.top+=1
            stack.list.append()
    def popstack(self):
        if self.top==self.bottom:
            return f"stack is underflow"
        elif (self.top!=self.bottom):
            self.top-=1
            stack.list.pop()
    def printstack(self):
        print("stack = ")
        print(stack.list)
p=stack(4)
p.pushstack()
p.pushstack()
p.pushstack()
    
